### PR TITLE
Document Update: Add an example to load config in shell

### PR DESCRIPTION
**Pull request by lff0305**
Document Update：Example for using shell script with curl to fetch config with Security Key enabled.

No code updated.

## What's the purpose of this PR

Document update: using curl to fetch config with Security Key enabled.

## Which issue(s) this PR fixes:
Fixes # NA.

## Brief changelog

4606648 Add shell script to fetch from config server.

### CODE REVIEW

**Comment by lff0305** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Anilople** [comment]
Please Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

File: docs/zh/usage/other-language-client-user-guide.md [hunk 1]
```diff
@@ -162,6 +162,23 @@ Apollo从1.6.0版本开始增加访问密钥机制，从而只有经过身份验
 |---------------|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Authorization | Apollo ${appId}:${signature}                   | appId: 应用的appId，signature：使用访问密钥对当前时间以及所访问的URL加签后的值，具体实现可以参考[Signature.signature](https://github.com/ctripcorp/apollo/blob/aa184a2e11d6e7e3f519d860d69f3cf30ccfcf9c/apollo-core/src/main/java/com/ctrip/framework/apollo/core/signature/Signature.java#L22)  |
 | Timestamp     | 从`1970-1-1 00:00:00 UTC+0`到现在所经过的毫秒数 | 可以参考[System.currentTimeMillis](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()) |
+### 1.5.1 例子，使用shell脚本从开启访问密钥的config server读取配置。需要`curl` 和 `openssl`
+```
```
Comment (Anilople) [inline docs/zh/usage/other-language-client-user-guide.md:166]:
```suggestion
```bash
```

File: docs/zh/usage/other-language-client-user-guide.md [hunk 2]
```diff
@@ -162,6 +162,23 @@ Apollo从1.6.0版本开始增加访问密钥机制，从而只有经过身份验
...
+appid="app1"                                                  // 修改需要读取的appid
```
Comment (Anilople) [inline docs/zh/usage/other-language-client-user-guide.md:167]: comment in bash is start with `#`.

How about change to 
```suggestion
# 需要读取的appid
appid="app1"
```

**Review by Anilople — CHANGES_REQUESTED** [review]

**Comment by nobodyiam** [comment]
as 1.9.0 was released, please update against the latest CHANGES.md, thanks!

File: CHANGES.md [hunk 3]
```diff
@@ -5,6 +5,78 @@ Release Notes.
 Apollo 1.10.0
 
 ------------------
+* [extend DataChange_CreatedBy, DataChange_LastModifiedBy from 32 to 64.](https://github.com/ctripcorp/apollo/pull/3552)
```
Comment (nobodyiam) [inline CHANGES.md:8]: It seems these are items in 1.9.0 release note...

File: CHANGES.md [hunk 4]
```diff
@@ -16,6 +16,7 @@ Apollo 1.10.0
 * [refactor: let open api more easier to use and development](https://github.com/apolloconfig/apollo/pull/3943)
 * [feat(scripts): use bash to call openapi](https://github.com/apolloconfig/apollo/pull/3980)
 * [Support search by item](https://github.com/apolloconfig/apollo/pull/3977)
+* [Document: Load configurations from shell scriopt](https://github.com/apolloconfig/apollo/pull/3853)
```
Comment (nobodyiam) [inline CHANGES.md:19]:
```suggestion
* [Document: Load configurations from shell script](https://github.com/apolloconfig/apollo/pull/3853)
```

File: docs/zh/usage/shell-user-guide.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+### 使用Shell Script读取Apollo的配置
```
Comment (Anilople) [inline docs/zh/usage/shell-user-guide.md:1]: rename to `shell-sdk-user-guide.md`?

File: docs/zh/usage/other-language-client-user-guide.md [hunk 6]
```diff
@@ -162,6 +162,8 @@ Apollo从1.6.0版本开始增加访问密钥机制，从而只有经过身份验
 |---------------|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Authorization | Apollo ${appId}:${signature}                   | appId: 应用的appId，signature：使用访问密钥对当前时间以及所访问的URL加签后的值，具体实现可以参考[Signature.signature](https://github.com/ctripcorp/apollo/blob/aa184a2e11d6e7e3f519d860d69f3cf30ccfcf9c/apollo-core/src/main/java/com/ctrip/framework/apollo/core/signature/Signature.java#L22)  |
 | Timestamp     | 从`1970-1-1 00:00:00 UTC+0`到现在所经过的毫秒数 | 可以参考[System.currentTimeMillis](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()) |
+### 1.5.1 例子：通过 Shell 脚本从 Config Server 读取配置,以及如何使用密钥访问
+参见[shell 脚本例子](./shell-user-guide.md)
```
Comment (Anilople) [inline docs/zh/usage/other-language-client-user-guide.md:166]: 删除这里的内容，通过_sidebar.md添加shell客户端的目录，会不会更好一些

例如添加一行**shell客户端使用指南**到 https://github.com/apolloconfig/apollo/blob/master/docs/zh/_sidebar.md
```markdown
- 使用文档
  - [Apollo使用指南](zh/usage/apollo-user-guide.md)
  - [Java客户端使用指南](zh/usage/java-sdk-user-guide.md)
  - [Shell客户端使用指南](zh/usage/shell-sdk-user-guide.md)
  - [.Net客户端使用指南](zh/usage/dotnet-sdk-user-guide.md)
  - [Go、Python、NodeJS、PHP等客户端使用指南](zh/usage/third-party-sdks-user-guide.md)
  - [其它语言客户端接入指南](zh/usage/other-language-client-user-guide.md)
```

这样在 左边侧边栏 目录就可以看到 Shell客户端使用指南 了

**Review by Anilople — CHANGES_REQUESTED** [review]

**Pull request closed without merging**